### ISSUE TITLE
The Script isn't working

### ISSUE THREAD

**Issue by Tuhin-thinks** [harshibar/common-intern#15]
### List of misleads:
- apply.py is supposed to get apply links from glassdoor.co.in, but it doesn't apply to **any** job!
- It does not upload any resume in PDF format. (there's no use of the PDF for grasshopper links)
- It cannot handle pop-ups that may arise anytime after the first search.
- There's no way to apply 1000s of jobs at a time, when there’s only function for *level* and *grasshopper* jobs.

Basically, it's a toy project and nothing serious in it and 
in-appropriate for use in any scenario.

_Most importantly, It can't apply 1000s of jobs, not even 10 jobs_
I doubt if she has tested it for direct links from the Glassdoor website.
